### DISCUSSION TITLE
Noobie install - cannot install on Pi4 - script termination

### DISCUSSION THREAD

**Discussion by Cheshire-astronomy** [aaronwmorris/indi-allsky#662] [Q&A]
I've tried to install it several times now, I choose the camera and the location but I keep getting this error

The following package was automatically installed and is no longer required:
  libfuse2
Use 'sudo apt autoremove' to remove it.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package indi-full
E: Unable to locate package indi-webcam
E: Unable to locate package indi-asi
E: Unable to locate package libasi
E: Unable to locate package indi-qhy
E: Unable to locate package libqhy
E: Unable to locate package indi-playerone
E: Unable to locate package libplayerone
E: Unable to locate package indi-sv305
E: Unable to locate package libsv305
E: Unable to locate package libaltaircam
E: Unable to locate package libmallincam
E: Unable to locate package libmicam
E: Unable to locate package libnncam
E: Unable to locate package indi-toupbase
E: Unable to locate package libtoupcam
E: Unable to locate package indi-gphoto
E: Unable to locate package indi-sx

The script exited abnormally, please try to run again...

**Comment by aaronwmorris** [comment]
What board and distribution are you using?

**Reply by Cheshire-astronomy** [reply]
Pi4 B - 2Gb & version 20230214 (Bullseye 32bit I think)

**Reply by aaronwmorris** [reply]
Try the 64bit release.  The astroberry indi repo is not complete for 32bit bullseye
